src/retention-management/nuxeo-retention-technical-overview.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Technical Overview
+description: Discover technical implementation of the Nuxeo Retention Management add-on.
+review:
+    comment: ''
+    date: '2019-10-17'
+    status: 'ok'
+labels:
+    - lts2019-wip
+    - jaubenque
+    - mlumeau
+    - retention-management
+toc: true
+tree_item_index: 100
+private: true
+---
+
+## Amazon S3 Compliance mode integration
+
+### {{> anchor 's3-configuration-requirements'}} Amazon S3 configuration requirements
+
+Compliance with SEC Rule 17a-4 requires to configure Nuxeo Platform to utilize Amazon S3 Compliance Buckets to store books and records required by regulation, thereby establishing the foundation for meeting the requirements of the Rule. Specifically,
+
+- Direct writes to the Amazon S3 storage system must be disabled, to ensure all documents transit through Nuxeo Platform for compliant processing.
+
+- The Amazon S3 Object Lock feature must be enabled in Compliance Mode on the bucket intended to store objects that are marked as final records.
+
+- Amazon S3 Versioning must be enabled.
+
+- The default retention value for Amazon S3 Compliance Buckets intended to retain compliant record objects must be set to zero (0).
+
+- No Min/Max range should be established for Amazon S3 Compliance Buckets intended to retain compliant record objects.
+
+- Amazon S3 Lifecycle Policies must not be configured for use within the Nuxeo Platform storage subsystem.
+
+
+### Amazon S3 buckets
+
+Nuxeo Platform with Nuxeo Retention Management add-on requires the usage of 2 Amazon buckets:
+
+- A standard S3 bucket as for any other standard Nuxeo instance,
+
+  - This bucket is used to store the standard documents,
+
+
+- A S3 bucket dedicated to the records that is configured on [Compliance mode]({{page page='nuxeo-retention-technical-overview'}}#s3-configuration-requirements):,
+
+  - This bucket is used to store the records only,
+  - Compared to the standard bucket, the Garbage Collector must be disabled as there is a [specific deletion process]({{page page='nuxeo-retention-technical-overview'}}#record-deletion-flow),
```

Comment:
Not sure we need to talk about GC, this is an implementation detail and it'll be dealt with automatically by the Nuxeo config.